.github/workflows/build_and_test.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         DATABASE_PORT: 5432
         DATABASE_NAME: postgres
         DATABASE_SECRET: '{"username":"postgres","password":"password"}'  # pragma: allowlist secret
+        SEED_SYSTEM_DATA: false
```

Comment:
I suspect this will also need to be set in the jobs that run migrations in environments which probably assume a truthy value if they don't set an explicit test env (which I hope they don't!)